core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_prowconfig.yaml

```diff
@@ -0,0 +1,154 @@
+tide:
+  queries:
+  - includedBranches:
+    - community-4.6
+    - community-4.7
+    - openshift-4.10
+    - openshift-4.11
+    - openshift-4.12
+    - openshift-4.13
+    - openshift-4.14
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-4.0
+    - release-4.10
+    - release-4.11
+    - release-4.12
+    - release-4.13
+    - release-4.14
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
```

Comment:
Why all these branches for a new repository?

Reply:
Well, for the cluster-dns-operator they make sense as the operator existed from the very first version of OCP 4. We don't have these branches for the new plugin. This may be misleading keeping the old release branches like if the plugin was delivered for those releases too or like it has those branches. For the new releases though, I suppose that ART automation will add new branches.

Reply:
Btw this config is for the backports as well as the config right after it (`openshift-4.15`). It feels a little premature to add the CI config for the backports while we don't have the initial release yet. For the moment, we cannot say which releases we may need to backport something to. I think I'd prefer to keep the CI config to the necessary minimum - `main` branch only, and add new things when we have need them. WDYT?

Reply:
Makes sense. We can always update the file as and when required. I will make the appropriate changes.